\documentclass[10pt]{article}
%\usepackage[10pt]{extsizes}
\usepackage[margin=1.25in]{geometry}

%\usepackage{times}
\usepackage[symbol]{footmisc}
\usepackage{url}
\usepackage{hyperref}
\usepackage{enumitem}
\usepackage{titlesec}
\usepackage{comment}
\usepackage{indentfirst}
\usepackage{multicol}
\usepackage{multirow}
\usepackage{etaremune}
\usepackage[normalem]{ulem}
\usepackage{float}

\usepackage{fontspec}
\setmainfont[Path=/usr/local/share/fonts/truetype/calibri/,
    BoldItalicFont=calibriz.ttf,
    BoldFont      =calibrib.ttf,
    ItalicFont    =calibrii.ttf]{calibri.ttf}
    
\setcounter{secnumdepth}{0}
\setlength{\parindent}{2em}
\setlength{\parskip}{0.02in}
\renewcommand{\baselinestretch}{1}

\titleformat{name=\section}[display]
    {\filright\scshape}{\thesection}{1em}{}
\titlespacing*{\section}{-5pt}{5pt}{5pt}
%\renewcommand{\thefootnote}{\fnsymbol{footnote}}

\hypersetup{
colorlinks=true,
urlcolor=cyan
}
\newcommand{\makerule}{\rule{\linewidth}{0.1mm}\par\vspace{0.02in}}

\let\olditemize\itemize
\renewcommand{\itemize}{
\olditemize
\setlength{\itemsep}{0.01in}
}
\let\oldenumerate\enumerate
\renewcommand{\enumerate}{
\oldenumerate
\setlength{\itemsep}{0.01in}
}
\setlist[itemize]{leftmargin=4em}
\setlist[enumerate]{leftmargin=4em}
%\setlist[etaremune]{leftmargin=4em}
\setlist[description]{leftmargin=7em}
\newlist{myitemize}{itemize}{1}
\setlist[myitemize,1]{label=-,leftmargin=2em}
\newlist{myitemize2}{itemize}{1}
\setlist[myitemize2,1]{leftmargin=-3em}


\begin{document} 

\begin{center}
{\large\bf 
CURRICULUM VITAE\\
Jing Ma, Ph.D.\\
}

September 1st, 2021 

\vspace{1em}

\end{center} 

\section{Personal Data}

\begin{table}[H]
\hskip0.4cm\begin{tabular}{p{2cm}p{12.5cm}}
%\begin{tabular}{ll}
Place of Birth & Henan, China                           \\
{Work Address}   & Fred Hutchinson Cancer Research Center \\
& Division of Public Health Sciences\\
& 1100 Fairview Ave. N\\
& PO Box 19024 - M2-B500 \\
& Seattle, WA 98109-1024 \\
Email          & jingma@fredhutch.org \\
%{Work Address}   & Texas A\&M University \\
%& Department of Statistics\\
%& 3143 TAMU \\
%& College Station, TX 77843 \\
%Email          & jingma@tamu.edu \\
Website        & \url{http://drjingma.com/}                          
\end{tabular}
\end{table}


\section{Education}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12.5cm}}
2006-2010 & Fudan University, Shanghai, China, B.S., Mathematics \\
& with University Distinction (Highest)\\
2010-2015 & University of Michigan, Ann Arbor, MI, Ph.D., Statistics \\
& {Advisor}: Prof. George Michailidis
\end{tabular}
\end{table}

%\begin{itemize}
%\item Fudan University, Shanghai, China, B.S., Mathematics, 2010 
%\vspace{-0.05in}
%\begin{myitemize}
%\item[] with University Distinction (Highest)
%\end{myitemize}
%\item University of Michigan, Ann Arbor, MI, Ph.D., Statistics, 2015
%\vspace{-0.05in}
%\begin{myitemize}
%\item[] {Advisor}: Prof. George Michailidis
%%\item[] {\em Dissertation Title}: 
%%Estimation and Inference in High-Dimensional Gaussian Graphical Models  with Structural Constraints
%\end{myitemize}
%\end{itemize}

\section{Postgraduate Training} 
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2015-2017 & Postdoctoral Research Fellow, {Department of Biostatistics and Epidemiology} \& {Department of Statistics}, University of Pennsylvania. %(08/15 - 07/17)
\end{tabular}
\end{table}


\section{Faculty Positions} 
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2017- & Assistant Professor of Biostatistics, Division of Public Health Sciences, Fred Hutchinson Cancer Research Center. \\
2019- & Affiliate Assistant Professor, Department of Biostatistics, University of Washington.  \\
2019-2020  & Assistant Professor, Department of Statistics, Texas A\&M University.\\
2020- & Adjunct Assistant Professor, Department of Statistics, Texas A\&M University.
\end{tabular}
\end{table}

%\begin{itemize}  
%\item Postdoctoral Research Fellow, {Department of Biostatistics and Epidemiology} \& {Department of Statistics}, University of Pennsylvania, 08/15 - 07/17
%\item Assistant Member of Biostatistics, Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, 08/17 - 08/19
%\begin{myitemize}
%\item  Affiliated Assistant Professor, Department of Biostatistics, University of Washington, 01/19 - 08/19
%\end{myitemize}
%\item Assistant Professor, Department of Statistics, Texas A\&M University, 09/19 - 
%\end{itemize}

\section*{Research Funding }

\textbf{A. Current projects as Principal Investigator}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2021-2022 & Fred Hutch Pathogen-Associated Malignancies Integrated Research Center / Microbiome Research Pilot Award  \\
& Title: Statistical Methods for Network-based Analysis of the Colorectal Cancer Microbiome \\
%& Role: Principal-Investigator\\
\end{tabular}
\end{table}


\textbf{B. Current projects as Co-Investigator}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2018-2022 & NIH R01 Award (PI: M. C. Wu)  \\
& Title: Joint Analysis of Microbiome and Other Genomic Data Types\\
%& Role: Co-Investigator (PI: M. C. Wu)\\
2018-2023 & NIH U19 Award (PI: D. Promislow) \\
& Title: The Dog Aging Project: The Genetic and Environmental Determinants of Healthy Aging in Companion Dogs\\
%& Role: Co-Investigator \\
2020-2024 & NIH R01 Award (PI: A. Shojaie) \\
& Title: Novel Statistical Inference for Biomedical Big Data\\
%& Role: Co-Investigator 
\end{tabular}
\end{table}


\textbf{C. Pending}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2022-2027 & NIH R01 Award \\
& Title: Statistical Methods for Network-based Integrative Analysis of Microbiome Data \\
& Role: Principal-Investigator
\end{tabular}
\end{table}
%\begin{table}[H]
%\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
%2021-2022 & Johnson \& Johnson WiSTEM2D Scholars Award Program  \\
%& Title: Statistical Modeling to Enhance Microbiome Biomarker Discovery\\
%& Role: Principal-Investigator\\
%\end{tabular}
%\end{table}
%
%\begin{table}[H]
%%\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
%\end{tabular}
%\end{table}


 \textbf{D. Completed}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2017-2018 & NIH R21 Award\\
&		Title: DMPA Use and Vaginal Bacterial Diversity among African Women\\
%&		FTE: 4\%\\
&		Role: Co-Investigator (PI: A. Roxby; Sub: D. Fredricks)\\
2018-2019 & NIH R01 Award\\
&	Title: The Impact of Prenatal Exposure to Persistent Organic Pollutants on Kinetics of Immune Response to Vaccines and Seroprotection in Infants\\
%& FTE: 10\% \\
& Role: Co-Investigator (PI: P. Bhatti) 
\end{tabular}
\end{table}

\section{Bibliography}

[* indicates alphabetical ordering authorship.]

[$\dagger$ indicates corresponding author.]

[Name underlined indicates student or postdoc mentored.]

\medskip

\textbf{A. Publications in Peer-Refereed Journals}

%\begin{etaremune}[leftmargin=4em]
\begin{enumerate}
\item \textbf{Ma, J}, Shojaie, A and Michailidis, G. Network-based pathway enrichment analysis with incomplete network information. {\em Bioinformatics}. 32(20):3165--3174, 2016.
\item \textbf{Ma, J} and Michailidis, G. Joint structural estimation of multiple graphical models. {\em Journal of Machine Learning Research}. 17:1--48, 2016.
\item von Rundstedt, F, Kimal, R, \textbf{Ma, J}, Arnold, J, Gohlke, J, Putluri, V, Krishnapuram, R, Piyarathna, D, Lotan, Y, Godde, D, Roth, S, Storkel, S, Levitt, J, Michailidis, G, Lerner, S, Coarfa, C, Sreekumar, A, Putluri, N. Integrated pathway analysis of a metabolic signature in bladder cancer - a linkage to The Cancer Genome Atlas project and prediction of survival. {\em Journal of Urology}. 195(6):1911--1919, 2016.
\item *Cai, TT, \textbf{Ma, J} and Zhang, L. CHIME: clustering of high-dimensional Gaussian mixtures with EM algorithm and its optimality. {\em Annals of Statistics}. 47(3):1234--1267, 2019. 
\begin{myitemize}
\item  {L. Zhang was a recipient of ASA Biopharmaceutical Section Student Paper Award at the 2017 ICSA Applied Statistics Symposium.}
\end{myitemize}
 \item *Cai TT, Li H, \textbf{Ma J}$\dagger$, and Xia Y. Differential Markov random field analysis with applications to detecting differential microbial community networks. {\em Biometrika}. 106(2):401--416, 2019. 
 \item \textbf{Ma J}, Karnovsky A, Afshinnia F, Wigginton J, Feldman H, Rader D, Shama K, Porter A, Rahman M, He J, Hamm L, Shafi T, Pennathur S, Michailidis G. Differential network-based enrichment analysis of lipid pathways altered in Chronic Kidney Disease progression. {\em Bioinformatics}. 35(18):3441--3452, 2019.
 \item \textbf{Ma J}{$\dagger$}, Shojaie A and Michailidis G. A comparative study of topology-based pathway enrichment analysis methods. {\em BMC Bioinformatics}. 20 (546). 2019 
 \item \underline{Wang Y}, Randolph T, Shojaie A and \textbf{Ma J}{$\dagger$}. The generalized matrix decomposition biplot and its application to the microbiome data. {\em mSystems}. 4:e00504-19. 2019.
\begin{myitemize}
\item  Selected as \textbf{Editor's pick}.
\end{myitemize}
 \item Vantaku V, Putluri V, Bader D, Maity S, \textbf{Ma J}, $\cdots$, Sreekumar A and Putluri N. Epigenetic loss of AOX1 expression via EZH2 leads to metabolic deregulation in bladder cancer. {\em Oncogene}. 39:6265--6285, 2020.   
 \item \textbf{Ma J}. Joint microbial and metabolite network estimation with the censored Gaussian graphical model. {\em Statistics in Biosciences}. 13:351--372, 2021.
\item Hellstern, M, \textbf{Ma, J}, \underline{Yue, K} and Shojaie, A. netgsa: Fast computation and interactive visualization for topology-based pathway enrichment analysis. {\it PLoS Computational Biology}. 2021.
\end{enumerate}

\textbf{B. Book Chapters}
\begin{enumerate}
\item Li, H and \textbf{Ma, J}. Graphical models in genetics, genomics and metagenomics. In {\em Handbook of Graphical Models.} Editors: Mathias Drton, Steffen Lauritzen, Marloes Maathuis, Martin Wainwright. Chapman \& Hall / CRC, 2018.
\item \textbf{Ma, J}, \underline{Yue, K} and Shojaie, A. Networks for compositional data. In \emph{Statistical Analysis of Microbiome Data.} Editors: Subharup Guha, Somnath Datta. Springer, 2020
\end{enumerate}

\textbf{C. Other Refereed Scholarly Publications}
\begin{enumerate}
\item \textbf{Ma, J.} Estimation and Inference in High-Dimensional Gaussian Graphical Models  with Structural Constraints. University of Michigan. 2015. [PhD Thesis]
\end{enumerate}

\textbf{D.  Preprints}
\begin{enumerate}
\item \underline{Yue, K}, \textbf{Ma, J}, Thornton, T and Shojaie, A. REHE: fast variance components estimation for linear mixed models. {\it Genetic Epidemiology.} Minor revision submitted, 2021. \url{https://www.biorxiv.org/content/10.1101/2021.02.03.429643v1}
\begin{myitemize}
\item Kun Yue was a recipient of the Best Student Paper Award at the 2021 WNAR meeting. 
\end{myitemize}
\item \underline{Wang Y}, Shojaie A, Randolph T and \textbf{Ma J}. Generalized matrix decomposition regression: estimation and inference for two-way structured data. {\it Submitted to Journal of the Royal Statistical Society, Series B (Statistical Methodology).} \url{https://arxiv.org/abs/2104.08408}
\item \underline{Wang Y}, \textbf{Ma J} and Shojaie A. Direct estimation of differential Granger causality between two high-dimensional time series. {\it Submitted to Biometrika.} \url{https://arxiv.org/abs/2109.07609}
\end{enumerate}

%\textbf{E. Working Papers}
%\begin{enumerate}\setcounter{enumi}{4}
%\item \underline{Wang Y}, Shojaie A, and \textbf{Ma J}. Direct estimation of differential granger causal relations between two high-dimensional time series. 
%\item \underline{Wang Y}, \textbf{Ma J}, Shojaie A, and Randolph T. Multi-view regression via JIVE. 
%%\item \underline{Wang Y}, Shojaie A, and \textbf{Ma J}. Extensions of DEGraph. 
%%\item \underline{Wang Y}, Randolph T, Shojaie A, and \textbf{Ma J}. Review on supervised data integration methods with applications to microbiome data analysis. 
%\item \textbf{Ma, J}, Roy, S and Michailidis, G. Fast estimation of change points in high-dimensional regime switching Markov random fields.
%\end{enumerate}
%


\section*{Oral Presentations}

\textbf{A. Invited Talks at Conferences and Symposia}
\begin{enumerate}  
\item[06/14] International Chinese Statistical Association/Korean International Statistical Society Joint Applied Statistics Symposium, Portland, OR \\
Title: {``Pathway enrichment analysis with incomplete network information."} 
\item[06/16] International Chinese Statistical Association Applied Statistics Symposium, Atlanta, GA \\
Title: {``A zero-inflated Poisson model for abundance quantification from metagenomic data."}   
\item[06/17] International Chinese Statistical Association Applied Statistics Symposium, Chicago, IL\\
Title: {``Differential network biology: testing differences in microbial networks."} 
%Title: ``A testing framework for detecting differential microbial communities."
\item[03/18] Fred Hutch Microbiome Symposium, Seattle, WA \\
Title: {``Inferring differential microbial interactions from co-occurrence data using Markov networks."} 
\item[06/18] Data Science Innovation Lab: Mathematical Challenges of Single Cell Dynamics, Bend, OR\\
Title: {``Understanding therapeutic failures through 4D single-cell analysis of metabolic heterogeneity."} 
\item[07/18] The 12th International Vilnius Conference on Probability Theory and Mathematical Statistics / 2018 IMS Annual Meeting on Probability and Statistics, Vilnius, Lithuania  \\
Title:  {``Mixed graphical models for joint analysis of microbiome and metabolomic data."}
\item[07/18] Joint Statistical Meetings, Vancouver, Canada; id.
\item[12/18] The 11th International Conference of the ERCIM WG on Computational and Methodological Statistics, Pisa, Italy  \\
Title: {``Sparse clustering of Gaussian mixtures."} 
\item[02/19] The Role of Genomics and Metagenomics in Human Health: Recent Developments in Statistical and Computational Methods, Banff, Canada\\
Title: {``Variance components estimation in linear mixed models."} 
\item[07/19] International Chinese Statistical Association Conference, Tianjin, China\\
Title: {``A framework for multi-view analysis of microbiome data."} 
\item[12/19] The 12th International Conference of the ERCIM WG on Computational and Methodological Statistics, London, UK \\
Title: {``Generalized matrix decomposition: From exploratory analysis to high-dimensional inference."}
%\item (Cancelled) Conference on Statistical Learning and Data Science, University of California, Irvine, US. (05/20)
\item[08/20] Joint Statistical Meetings, Philadelphia, PA  (Virtual) \\
Title: {``Fast estimation of change points in regime switching Gaussian graphical models."} 
\item[07/21] International Chinese Statistical Association Conference, Xi'an, China; (rescheduled to 07/22)
%\item ICSA/KISS Joint Applied Statistics Symposium, Portland, OR; {\em Pathway enrichment analysis with incomplete network information.} (06/14)
%\item ICSA Applied Statistics Symposium, Atlanta, GA; {\em A zero-inflated Poisson model for abundance quantification from shotgun metagenomic data.}  (06/16) 
%\item ICSA Applied Statistics Symposium, Chicago, IL; {\em A testing framework for detecting differential microbial communities.} (06/17)
%\item Fred Hutch Microbiome Symposium, Seattle, WA; {\em Inferring differential microbial interactions from co-occurrence data using Markov networks.} (03/18) 
%\item Data Science Innovation Lab: Mathematical Challenges of Single Cell Dynamics, Bend, OR; {\em Understanding therapeutic failures through 4D single-cell analysis of metabolic heterogeneity.} (06/18)
%\item 12th International Vilnius Conference on Probability Theory and Mathematical Statistics / 2018 IMS Annual Meeting on Probability and Statistics, Vilnius, Lithuania; {\em Mixed graphical models for joint analysis of microbiome and metabolomic data.} (07/18)
%\item Joint Statistical Meetings, Vancouver, Canada; {\em id.} (07/18)
%\item CMStatistics Conference, Pisa, Italy; {\em Sparse clustering of Gaussian mixtures.} (12/18)
%\item The Role of Genomics and Metagenomics in Human Health: Recent Developments in Statistical and Computational Methods, Banff, Canada; {\em Variance components estimation in linear mixed models.} (02/19)
%\item ICSA China Conference, Tianjin, China; {\em A framework for multi-view analysis of microbiome data.} (07/19)
%\item University of Michigan 50th Anniversary Symposium, Ann Arbor, US; {\em Graphical models and differential networks for microbiome data.} (09/19)
%\item CMStatistics Conference, London, UK; {\em Generalized matrix decomposition: From exploratory analysis to high-dimensional inference.} (12/19)
%\item (Cancelled) Conference on Statistical Learning and Data Science, University of California, Irvine, US. (05/20)
%\item (Online) Joint Statistical Meetings, Philadelphia, PA; {\em Fast estimation of change points in regime switching Gaussian graphical models}. (08/20)
%\item CMStatistics Conference, London, UK; (12/20)
\end{enumerate}

\textbf{B. Invited Seminars and Colloquia}
\begin{enumerate}\setcounter{enumi}{14}
\item[10/16] Department of Mathematics and Statistics, Lancaster University, Lancaster, UK\\
Title: {``Differential network biology: testing differences in microbial networks."}
\item[01/17] Department of Statistics, University of Warwick, Coventry, UK; id. 
\item[02/17] Biostatistics Program, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA; id.
\item[01/18] Department of Biostatistics, University of Washington, Seattle, WA; id.
\item[01/18] Department of Statistics, University of Florida, Gainesville, FL \\
Title: {``Graphical models and differential networks for microbiome data."} 
\item[01/18] Biostatistics/ATME Joint Seminar, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA\\
Title: {``Network-based enrichment analysis."} 
\item[02/18] Statistical Learning Applied to Biology Lab Seminar, Department of Biostatistics, University of Washington, Seattle, WA\\
Title: {``Statistical machine learning methods in genetics and genomics."} 
\item[02/18] Translational Research Program, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA\\
Title: {``Differential network biology."} 
\item[11/18] Department of Statistics, Texas A\&M University, College Station, TX\\
Title: {``Graphical models and differential networks for microbiome data."} 
\item[09/19] University of Michigan 50th Anniversary Symposium, Ann Arbor, US\\
Title: {``Graphical models and differential networks for microbiome data."} 
\item[12/19] (Virtual) The Dog Aging Project Science Seminar Series, Texas A\&M University, College Station, TX\\
Title: {``Statistical considerations in the Dog Aging Project."}
\item[10/20] (Virtual) Hanash Lab Meeting, The University of Texas MD Anderson Cancer Center\\
Title: {``Machine learning tools for omics data analysis."}
\item[01/21] (Virtual) Gut Origins of Latino Diabetes (GOLD) Monthly Meeting, Albert Einstein College of Medicine, New York, NY\\
Title: {``Statistical methods for microbiome data analysis."}
\item[03/21] (Virtual) Translational Data Science Seminar Series, Fred Hutchinson Cancer Research Center, Seattle, WA\\
Title: {``Systems biology approaches for microbiome data analysis."}
\item[11/21]  Biostatistics Seminar Series, Fred Hutchinson Cancer Research Center, Seattle, WA\\
Title: TBD.
\item[11/21]  ASA Section of Statistics in Genomics and Genetics (SSGG) Webinars\\
Title: TBD.
%\item Department of Mathematics and Statistics, Lancaster University, Lancaster, UK; {\em Differential network biology.} (10/16)
%\item Student Lunch Seminar, Department of Statistics, University of Pennsylvania; {\em Statistical methods for microbiome data.} (10/16) 
%\item Department of Statistics, University of Warwick, Coventry, UK; {\em Differential network biology: testing differences in microbial networks.} (01/17)
%\item Li Lab, Department of Biostatistics, University of Pennsylvania; {\em id.} (02/17)
%\item Biostatistics Program, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA; {\em id.} (02/17)
%\item Department of Biostatistics, University of Washington, Seattle, WA; {\em id.} (01/18)
%\item Department of Statistics, University of Florida, Gainesville, FL; {\em id.} (01/18)
%\item Biostatistics/ATME Joint Seminar, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA; {\em Network-based enrichment analysis.} (01/18)
%\item SLAB Lab Seminar, Department of Biostatistics, University of Washington, Seattle, WA; {\em Statistical machine learning methods in genetics and genomics.} (02/18)
%\item Postdoc Meeting, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA; {\em Statistical methods for microbiome data.} (02/18)
%\item Translational Research Program Faculty Meeting, Public Health Sciences Division, Fred Hutchinson Cancer Research Center, Seattle, WA; {\em Differential network biology.} (02/18)
%\item Department of Statistics, Texas A\&M University, College Station, TX; {\em Differential network biology: testing differences in microbial networks.} (11/18)
%\item DAP Science Seminar Series, Texas A\&M University, College Station, TX; {\em Statistical considerations in the Dog Aging Project.} (12/19)
\end{enumerate}
%
%\textbf{Contributed Oral presentations}
%\begin{itemize}  
%\item Joint Statistical Meetings, Chicago, IL. (08/16)
%\item Joint Statistical Meetings, Seattle, WA. (08/15)
%\item The 9th ICSA International Conference, Hong Kong, China. (12/13)
%\item Joint Statistical Meetings, Montreal, Canada. (08/13)
%\item Joint Statistical Meetings, San Diego, CA. (08/12)
%\end{itemize}
%
%\textbf{Poster Presentations}
%\begin{itemize}  
%\item ENAR Annual Meeting, Washington D.C. (03/17)
%\item Michigan Student Symposium for Interdisciplinary Statistical Sciences, Ann Arbor, MI. (04/12 \& 03/14)
%\end{itemize}



\section*{Awards, Honors and Scholarships}

\textbf{A. National Awards}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2008 & National Merit Scholarship from Ministry of Education of China  \\
2015 & Travel Award, National Science Foundation \\
2018 & Travel Award, Data Science Innovation Lab \\
2018 & Travel Award, The Jayne Koskinas Ted Giovanis Foundation for Health and Policy
\end{tabular}
\end{table}

\textbf{B. Department/University Awards}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2010 & College Graduate Excellence Award of Shanghai  \\
2011 & Outstanding First Year Ph.D. Student Award \\
2011-2012 & Rackham International Students Fellowship \\
2012-2013 & Outstanding Graduate Student Instructor Award\\
2012-2015 & Rackham School of Graduate Studies Conference Travel Grant
\end{tabular}
\end{table}

%\begin{itemize} 
%\item College Graduate Excellence Award of Shanghai, 2010 
%\item Outstanding First Year Ph.D. Student Award, 2011 
%\item Rackham International Students Fellowship, 2011-2012 
%\item Outstanding Graduate Student Instructor Award, 2012-2013
%\item Rackham School of Graduate Studies Conference Travel Grant, 2012, 2013, 2014 \& 2015 
%\end{itemize}

\section*{Professional Activities}

%\textbf{Editorial Service}
%\begin{itemize}
%\item  None.
%\end{itemize}
%
%\textbf{Grant Review}
%\begin{itemize}
%\item None.
%\end{itemize}
%
%\textbf{Invited Conference Sessions Organized}
%\begin{itemize}
%\item None.
%\end{itemize}

\textbf{A. Invited Session Organized}

%\begin{table}[H]
%\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
%2021 & ENAR, Baltimore, MD. 
%2021 & JSM, Seattle, WA.
%\end{tabular}
%\end{table}

\textbf{B. Session Chair}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2012 & {\em Methods for Variable and Model Selection -- Contributed Papers}, Joint Statistical Meetings, San Diego, CA. \\
2016 & {\em Efficient Methods for Structured Large Genomics Data -- Contributed Papers}, Joint Statistical Meetings, Chicago, IL. \\
2017 & {\em  Understanding the Microbiome Complexity: Genetics and Networks -- Invited Papers}, ICSA Applied Statistics Symposium, Chicago, IL. \\
2019 & {\em Methodological Innovation in Large-scale Genomics Data Analysis -- Invited Session}, ICSA China Conference, Tianjin, China. 
\end{tabular}
\end{table}

\textbf{C. Referee Service} 
\begin{itemize}
\item[] Referee for 
Annals of Applied Statistics,
Bioinformatics, 
Biometrika, 
Biometrical Journal, 
Biometrics, 
Biostatistics, 
Electronic Journal of Statistics, 
Frontiers in Genetics, 
IEEE Access, 
Journal of Computational Biology, 
Journal of the American Statistical Association -- Applications \& Case Studies, 
Journal of the American Statistical Association -- Theory \& Methodology, 
Journal of Machine Learning Research, 
Journal of Multivariate Analysis, 
Molecular \& Cellular Proteomics, 
Optimization and Engineering, 
PLoS Computational Biology, 
Scientific Reports (Nature), 
Statistics in Biosciences, 
Statistics in Medicine,
Wiley Interdisciplinary Reviews: Computational Statistics 
\end{itemize}

\textbf{D. Conference Review}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2016 & Reviewer for  International Conference on Information Systems \\
2016 & Reviewer for  Conference on Neural Information Processing Systems (NIPS)\\
2018 & Judge for  ASA Section on Genomics and Genetics Student Paper Competition\\ 
2018 & Reviewer for  Conference on Neural Information Processing Systems (NIPS) \\
2019 & Judge for  ASA Section on Genomics and Genetics Student Paper Competition \\
2020 & Judge for  ASA Section on Genomics and Genetics Student Paper Competition \\
\end{tabular}
\end{table}

\textbf{E. Grant Review}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2020 & FHCRC Translational Data Science Integrated Research Center Pilot Grant \\
2020 & UW ITHS Research Innovation Award \\
2020 & FHCRC Division of Public Health Sciences Bid \& Proposal Projects\\
2021 & FHCRC Translational Data Science Integrated Research Center Pilot Grant
\end{tabular}
\end{table}

\textbf{F. Professional Organizations}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2010- & American Statistical Association\\
2014- & International Chinese Statistical Association\\
2016- & Eastern North American Region International Biometric Society
\end{tabular}
\end{table}


\section*{Teaching Responsibilities}

\textbf{A. Instructor at the University of Michigan}
\begin{itemize} 
\item STAT250: Introduction to Statistics and Data Analysis (Lab Instructor). Fall 2010 \& Winter 2011
\item Applied Qualifying Exam. Summer 2012, 2013 \& 2014
\item Review of Linear Algebra. Summer 2013 \& 2014
\end{itemize}

\textbf{B. Graduate Student Instructor at the University of Michigan}
\begin{itemize}  
\item English Language Institute. Summer 2011
\item STAT600: Applied Statistics and Data Analysis. Fall 2011 \& 2012
\begin{myitemize}  
\item \uline{GSI Excellence in Teaching Award}
\end{myitemize}
\item STAT425: Introduction to Probability and Statistics. Fall 2011 \& Winter 2012
\item STAT601: Multivariate and Categorical Data Analysis. Winter 2012
\end{itemize}

\textbf{C. Instructor at Texas A\&M University}
\begin{itemize} 
\item STAT312: Statistics for Biology. Fall 2019
\end{itemize}

\section*{Advising and Mentoring}

\textbf{A. Postdoctoral Fellows}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2018-2020 & Yue Wang, PhD in Biostatistics, UNC Chapel Hill (joint with A. Shojaie).\\
& Current/First position: Assistant Professor of Statistics at Arizona State University.\\
2021 -  & Ilias Moysadis, PhD in Statistics, Pennsylvania State University. 
\end{tabular}
\end{table}

\textbf{B. Graduate Students}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2021- & Anh-Minh Nguyen, UW Biostatistics MS Capstone student.
\end{tabular}
\end{table}

\textbf{C. Student Collaborator}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2017-2018 & Nanxun Ma, UW Biostatistics PhD student, working with M. C. Wu.\\
2017-2020 & Kun Yue, UW Biostatistics PhD student, working with A. Shojaie.\\
2018-2020 & Michael Hellstern, UW Statistics MS student, working with A. Shojaie. \\
2020-2021 & Yunbi Nam, UW Biostatistics MS Capstone student. \\
2020- & Kristyn Pantoja, Texas A\&M Statistics PhD student, working with D. E. Jones. \\
2021- & Lakshin Kumar, UW Biochemistry undergraduate student, working with D. Promislow.
\end{tabular}
\end{table}

\textbf{D. MS and PhD Committees in Non-Chair Role}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2018-2019 & Arjun Sondhi, UW Biostatistics PhD student (advisor: A. Shojaie). \\
2019-2020 & Junsouk Choi, Texas A\&M Statistics PhD student (advisor: N. Yang). 
%2020- & Elizabeth Thompson, Texas A\&M Statistics PhD student (advisor: R. Carroll).
\end{tabular}
\end{table}

%\textbf{PhD Committees as Chair}
%\begin{itemize}
%\item None.
%\end{itemize}

%\section{Editorial Responsibilities}
%
%None.
%
%\section{Special National Responsibilities}
%
%None.
%\begin{table}[H]
%\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
%2021 & Organizer of ENAR invited session.
%\end{tabular}
%\end{table}

\section{Special Local Responsibilities}

\textbf{A. University of Michigan}

\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2011-2012 & Coordinator of Reading Group on Statistical Modeling and Analysis of Networks, Department of Statistics.\\
2011-2013 & Co-Chair, Graduate Student Statistical Topics Seminar Series, Department of Statistics.\\
2012-2013 & Student committee member of the Seventh Michigan Student Symposium for Interdisciplinary Statistical Sciences (MSSISS).
\end{tabular}
\end{table}



\textbf{B. Fred Hutchinson Cancer Research Center}
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2018 & Organizing committee member of the 2018 Fred Hutch Microbiome Symposium.\\
2018 & Faculty host for UW Biostatistics Prospective PhD Student Visit Days.\\
2019 & Chair, Biostatistics Seminar Series.\\
2021 & Faculty host for UW Biostatistics Prospective PhD Student Visit Days. 
\end{tabular}
\end{table}


\textbf{C. Texas A\&M University }
\begin{table}[H]
\hskip0.9cm\begin{tabular}{p{1.6cm}p{12cm}}
2019-2020 & Chair of grants writing committee.
\end{tabular}
\end{table}




\section*{Publicly Available Software}
\begin{itemize}
\item {\bf netgsa}: R package for network-based pathway enrichment analysis. 
\begin{myitemize}
\item Stable release on \href{https://cran.r-project.org/web/packages/netgsa/index.html}{CRAN}.
\item Development version on \href{https://github.com/mikehellstern/netgsa}{GitHub}.
\end{myitemize}
\item {\bf CHIME}: Matlab code for clustering high-dimensional Gaussian mixtures with the EM algorithm. On \href{https://github.com/drjingma/gmm}{GitHub}.
\item \textbf{DNEA}: R code for differential network-based enrichment analysis. On \href{https://github.com/wiggie/DNEA}{GitHub}.
\item {\bf GMDecomp}: R package for generalized matrix decomposition (GMD) and GMD-biplots. On \href{https://github.com/taryue/GMDecomp}{GitHub}.
\item \textbf{metaMint}: R package for joint estimation of metabolite and microbial interaction networks. On \href{https://github.com/drjingma/metaMint}{GitHub}.
%\begin{myitemize}
%\item Development version on \href{https://github.com/drjingma/metaMint}{GitHub}.
%\end{myitemize}
\item {\bf TestBMN}: R package for testing high-dimensional binary Markov networks. On \href{https://github.com/drjingma/TestBMN}{GitHub}.
%\begin{myitemize}
%\item Development version on \href{https://github.com/wiggie/DNEA}{GitHub}.
%\end{myitemize}
\end{itemize}


%\section{Mentorship Record}



%\begin{itemize}
%\item Research Assistant, { Department of Computational Medicine \& Bioinformatics}, University of Michigan, 1/2015 - 7/2015
%\item Research Assistant, { Department of Molecular \& Cellular Biology}, Baylor College of Medicine, 9/2014 - 5/2015
%\item Research Scientist Intern, {Avaya Labs}, 5/2013 - 7/2013 
%\item Student Consultant, {Center for Statistical Consultation and Research}, University of Michigan, 1/2013 -12/2013
%\end{itemize}

%\section*{References}
%% (next page)
%\begin{itemize} 
%\item Please see my references on the following page.
%\end{itemize}



%\newpage
\begin{comment}
\section*{References}
\begin{description}
\item[George Michailidis] Professor of Statistics and Director of the Informatics Institute, University of Florida
\begin{myitemize2}%[leftmargin=*]
\item[] 205 Griffin-Floyd Hall, P.O. Box 118545
\item[] Gainesville, FL 32611, USA 
\item[] (352) 294-3913
\item[] gmichail@ufl.edu 
\end{myitemize2}

\item[Hongzhe Li] Professor of Biostatistics and Statistics, University of Pennsylvania 
\begin{myitemize2}
\item[] 215 Blockley Hall, 423 Guardian Drive
\item[] Philadelphia, PA 19104, USA 
\item[] (215) 573-5038
\item[] hongzhe@upenn.edu
\end{myitemize2}

\item[T. Tony Cai] Dorothy Silberberg Professor of Statistics, University of Pennsylvania
\begin{myitemize2}
\item[] 469 Jon M. Huntsman Hall, 3730 Walnut Street 
\item[] Philadelphia, PA 19104, USA 
\item[] (215) 898-8224 
\item[] tcai@wharton.upenn.edu 
\end{myitemize2}

%\item[Kerby Shedden] Professor of Statistics, University of Michigan
%\begin{myitemize2}
%\item[] 461 West Hall, 1085 South University Avenue
%\item[] Ann Arbor, MI 48109, USA
%\item[] (734) 764-0438
%\item[] kshedden@umich.edu 
%\end{myitemize2}

%\item[Ali Shojaie] Associate Professor of Biostatistics, University of Washington
%\begin{myitemize2}
%\item[] F-642 Health Sciences Building, Box 357232 
%\item[] Seattle, WA 98195, USA
%\item[] (206) 616-5323 
%\item[] ashojaie@uw.edu 
%\end{myitemize2}
\end{description}
\end{comment}

\end{document}


